.swiftlint.yml

```diff
@@ -6,6 +6,7 @@ disabled_rules:
   - void_return
   - empty_parentheses_with_trailing_closure
   - comma
+  - trailing_whitespace
```

Comment:
Hygiene-wise, this is a good rule to keep — if Xcode is hassling you, make sure that you have these settings enabled:

<img width="800" alt="screen_shot_2019-01-02_at_10_51_06_am" src="https://user-images.githubusercontent.com/1198851/50599577-7a2cca00-0e7c-11e9-9e76-ab2c11c8e132.png">

Reply:
Sounds good, I'll drop this from the PR then.